src/frameworks/index.ts

```diff
@@ -107,7 +109,7 @@ const SupportLevelWarnings = {
   ),
 };
 
-export const FIREBASE_FRAMEWORKS_VERSION = "^0.7.0";
+export const FIREBASE_FRAMEWORKS_VERSION = "0.6.1-canary.35a57cb";
```

Comment:
This is in ^0.8.0 now

Reply:
Are we able to test this with experiments enabled on the Firebase CLI?

Reply:
Once merged and cut, yes. If you'd like to help verify the changes in this branch, you can install it manually via `npm i -g github:firebase/firebase-tools#jamesdaniels_nuxtFixes`